Sources/Build/BuildPlan.swift

```diff
@@ -867,10 +871,12 @@ public final class ProductBuildDescription {
             args += ["-emit-executable"]
         }
 
-        // On linux, set rpath such that dynamic libraries are looked up
-        // adjacent to the product. This happens by default on macOS.
+        // Set rpath such that dynamic libraries are looked up
+        // adjacent to the product.
         if buildParameters.triple.isLinux() {
             args += ["-Xlinker", "-rpath=$ORIGIN"]
+        } else if buildParameters.triple.isDarwin() {
+            args += ["-Xlinker", "-rpath", "-Xlinker", "@executable_path"]
```

Comment:
Although it technically won't matter in this specific case, these days you should use `@loader_path`.

Reply:
I switched it.